opentelemetry-quickstart/src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
 quarkus.application.name=myservice
-quarkus.otel.exporter.otlp.traces.endpoint=http://localhost:4317
+quarkus.opentelemetry.enabled=true
+quarkus.opentelemetry.metrics.enabled=true
```

Comment:
Please doublecheck the property names, they use otel, not opentelemetry.

Reply:
You're right, that was the issue, thanks!